src/api/print.api.js

```diff
@@ -121,12 +138,23 @@ class GeoAdminCustomizer extends BaseCustomizer {
         }
 
         if (anchor) {
-            symbolizer.graphicXOffset = adjustWidth(-anchor[0] * scale, this.printResolution)
-            symbolizer.graphicYOffset = adjustWidth(-anchor[1] * scale, this.printResolution)
+            symbolizer.graphicXOffset = symbolizer.graphicXOffset
+                ? adjustWidth(
+                      (size[0] / 2 - anchor[0] + symbolizer.graphicXOffset) * scale,
+                      this.printResolution
+                  )
+                : 0
+            // don't ask why it works, but that's the best I could do.
+
+            symbolizer.graphicYOffset = symbolizer.graphicYOffset
+                ? (symbolizer.graphicYOffset = adjustWidth(-size[1], this.printResolution))
+                : 0
         }
         if (size) {
             symbolizer.graphicWidth = adjustWidth(size[0] * scale, this.printResolution)
         }
+        symbolizer.graphicXOffset = symbolizer.graphicXOffset ? symbolizer.graphicXOffset : 0
+        symbolizer.graphicYOffset = symbolizer.graphicYOffset ? symbolizer.graphicYOffset : 0
```

Comment:
can be simplified to something like `symbolizer.graphicYOffset ?? 0`

Reply:
I was convinced somehow that the ?? operator was only for 'null', but it seems you're right and it works as well for undefined.